wdtk-rdf/src/main/java/org/wikidata/wdtk/rdf/RdfConverter.java

```diff
@@ -474,4 +493,23 @@ boolean hasTask(int task) {
 		return ((this.tasks & task) == task);
 	}
 
+	/**
+	 * Creates an URI which represents the statement rank in a triple.
+	 * 
+	 * @param rank
+	 * @return
+	 */
+	String rankToRdfRepresentation(StatementRank rank) {
```

Comment:
Rename to "getUriStringFromRank":
* "Create" should only be used for methods that create new objects where creation has a cost or implies ownership responsibility (note that here you don't even create a String at all)
* "RdfRepresentation" is too vague.